theano/sparse/tests/test_basic.py

```diff
@@ -2965,7 +2965,8 @@ def structured_function(*args):
     sparse.trunc,
     np.trunc,
     test_dtypes=[m for m in sparse.all_dtypes
-                 if not m in sparse.complex_dtypes])
+                 if not m in sparse.complex_dtypes],
+    grad_test=False)
```

Comment:
We can add another test that just checks that grad is always zero (because that is how it's defined).

But trying to use verify_grad and fiddling with the sampling is the wrong way to go I think.